#組み込み例外を発生させる
# raise文を使って例外を発生させる
value = 200

if value > 100:
    raise ValueError(valueは100以下で)
print("raise 文のあとは実行されません")


